querybook/server/app/auth/oauth_auth.py

```diff
@@ -109,9 +111,19 @@ def _fetch_access_token(self, code):
         return resp["access_token"]
 
     def _get_user_profile(self, access_token):
-        resp = requests.get(
-            self.oauth_config["profile_url"], params={"access_token": access_token}
-        )
+        if(self.oauth_config["token_place"] == 'headers'):
```

Comment:
can we make headers the default option actually? I doubt any oauth provider uses access_token from query params. This was legacy code from pinterest's internal oauth system that we didn't bother to fix 😢

Reply:
Removed in the last commit. By default, it would be in the headers.